concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-266	266	patear	kick	1135	KICK
+Lionnet-1985-364-267	267	pecho	chest	1592	CHEST
+Lionnet-1985-364-268	268	pedir	ask, request	1828	ASK (REQUEST)
+Lionnet-1985-364-269	269	peinarse	to comb	2070	COMB (VERB)
+Lionnet-1985-364-270	270	pelo, lana	hair, wool	2381	FEATHER OR FUR OR HAIR
+Lionnet-1985-364-271	271	pene	penis	1222	PENIS
+Lionnet-1985-364-272	272	perder en el juego	lose in a game	1660	LOSE
+Lionnet-1985-364-273	273	perro	dog	2009	DOG
+Lionnet-1985-364-274	274	perseguir	pursue	1047	PURSUE
+Lionnet-1985-364-275	275	pesado	heavy	1210	HEAVY
+Lionnet-1985-364-276	276	pescado	fish	227	FISH
+Lionnet-1985-364-277	277	picar	sting	571	STING
+Lionnet-1985-364-278	278	pie	foot	1301	FOOT
+Lionnet-1985-364-279	279	piedra	stone	857	STONE
+Lionnet-1985-364-280	280	piel	skin	763	SKIN
+Lionnet-1985-364-281	281	pierna, rodilla	leg, knee	1297	LEG
+Lionnet-1985-364-282	282	pino	pine	1740	PINE
+Lionnet-1985-364-283	283	piojo de la cabeza	head louse	310	HEAD LOUSE
+Lionnet-1985-364-284	284	pisar	tread	575	TREAD
+Lionnet-1985-364-285	285	pluma	feather	1201	FEATHER
+Lionnet-1985-364-286	286	poder	be able to	972	BE ABLE
+Lionnet-1985-364-287	287	podrirse, podre	rot	2907	ROT
+Lionnet-1985-364-288	288	prender lumbre	make fire	737	LIGHT (IGNITE)
+Lionnet-1985-364-289	289	pulga	flea	232	FLEA
+Lionnet-1985-364-290	290	punta	tip	2992	TIP (OF OBJECT)
+Lionnet-1985-364-291	291	quedarse, faltar	be left	987	REMAIN
+Lionnet-1985-364-292	292	quejarse	complain	3642	COMPLAIN
+Lionnet-1985-364-293	293	raíz	root	670	ROOT
+Lionnet-1985-364-294	294	rajarse	split	1437	SPLIT
+Lionnet-1985-364-295	295	rana	frog	503	FROG
+Lionnet-1985-364-296	296	raspar	scrape	1767	SCRAPE
+Lionnet-1985-364-297	297	rastrojo	stubble		
+Lionnet-1985-364-298	298	rata	rat	1490	RAT
+Lionnet-1985-364-299	299	ratón	mouse	1491	MOUSE
+Lionnet-1985-364-300	300	regalar	give gift		
+Lionnet-1985-364-301	301	regañar	scold	299	SCOLD
+Lionnet-1985-364-302	302	regresar	return	581	COME BACK
+Lionnet-1985-364-303	303	reírse	laugh	1355	LAUGH
+Lionnet-1985-364-304	304	renacuajo	tadpole		
+Lionnet-1985-364-305	305	robar	steal	713	STEAL
+Lionnet-1985-364-306	306	rodilla	knee	1371	KNEE
+Lionnet-1985-364-307	307	rojo	red	156	RED
+Lionnet-1985-364-308	308	saber	know	1410	KNOW (SOMETHING)
+Lionnet-1985-364-309	309	sal	salt	1274	SALT
+Lionnet-1985-364-310	310	salir	go out	1107	GO OUT
+Lionnet-1985-364-311	311	sangre	blood	946	BLOOD
+Lionnet-1985-364-312	312	sapo	toad	894	TOAD
+Lionnet-1985-364-313	313	secarse	to dry up	1786	DRY UP
+Lionnet-1985-364-314	314	seis	six	1703	SIX
+Lionnet-1985-364-315	315	sembrar	sow seeds	748	SOW SEEDS
+Lionnet-1985-364-316	316	semilla	seed	714	SEED
+Lionnet-1985-364-317	317	sentarse	sit down	1649	SIT DOWN
+Lionnet-1985-364-318	318	sentarse (suj. plur.)	sit down (several)		
+Lionnet-1985-364-319	319	sentir, desear	feel, desire	1079	FEEL
+Lionnet-1985-364-320	320	ser hondo	be deep	1593	DEEP
+Lionnet-1985-364-321	321	serpiente	snake	730	SNAKE
```

Comment:
Similar to cerro/loma, we have two different words for SNAKE. But again, the mappings are fine,